manifest.json

```diff
@@ -12,15 +12,15 @@
     },
```

Comment:
Signature for `...caUq`

`t+hoIWX+CF+EkHWa8IXI8eu/Q15Igdq2s/zM2u68J/G4cPl9VKNLBKPkjrhwPaQQecxveJk/ITMsFfpx2KbxBw==`